### DISCUSSION TITLE
[@types/react]: The `onClick` attribute does not exist in `DOMAttributes`, it has been renamed to `onMouse` in version 18.3.9.

### DISCUSSION THREAD

**Discussion by tientq64** [DefinitelyTyped/DefinitelyTyped#70697] [Issues with a @types package] [answered]
![image](https://github.com/user-attachments/assets/9c73c59d-d850-4491-b08e-a17f8c7c6f07)

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
Can you provide a reproduction? This doesn't look like types from `@types/react`

**Reply by tientq64** [reply]
This is a minimal reproduction. Run `bun install`, then open the `src/main.tsx` file, it will show an error on the `onClick` attribute, but not on `onMouse`.
https://github.com/tientq64/types-react-bug

**Reply by eps1lon** [reply, marked as answer]
This is an IDE bug then. `tsc` will compile happily and VSCode shows only and error on `onMouse` as expected:
![Screenshot 2024-09-30 at 10 42 11](https://github.com/user-attachments/assets/3c55dc82-ab51-4fbe-b6ea-6e4c44ad7336)

If you're also using VSCode, make sure it's configured to treat `.tsx` files as TypeScript

**Reply by tientq64** [reply]
I also use VSCode. I accidentally switched from `bun` to `npm`, and everything worked as expected. It seems the issue was related to file writing with `bun`. Issue solved, thanks for your help.
